packages/clerk-js/src/ui/contexts/components/SignIn.ts

```diff
@@ -35,14 +37,17 @@ export const useSignInContext = (): SignInContextType => {
   const { queryParams, queryString } = useRouter();
   const options = useOptions();
   const clerk = useClerk();
-  const isCombinedFlow = options.experimental?.combinedFlow;
 
   if (context === null || context.componentName !== 'SignIn') {
     throw new Error(`Clerk: useSignInContext called outside of the mounted SignIn component.`);
   }
 
-  const { componentName, mode, ..._ctx } = context;
-  const ctx = _ctx.__experimental?.combinedProps ? { ..._ctx, ..._ctx.__experimental?.combinedProps } : _ctx;
+  const isCombinedFlow =
+    Boolean(!options.signUpUrl && options.signInUrl && !isAbsoluteUrl(options.signInUrl)) ||
+    context.withSignUp ||
+    false;
```

Comment:
`false` isn't needed and it won't have an effect in this expression
```suggestion
    context.withSignUp
```

Reply:
It is needed since withSignUp is optional. https://github.com/clerk/javascript/actions/runs/12791340810/job/35659240319?pr=4788

Reply:
Yeah, I see that. I wonder if we really need to mark all of these context properties as required: https://github.com/clerk/javascript/blob/main/packages/clerk-js/src/ui/contexts/components/SignIn.ts#L14-L27 , but that's a discussion for another time